docs/CONTRIBUTING.html

```diff
@@ -8,7 +8,7 @@
     <title>Contributing to the Knowledge Centre </title>
     <meta name="viewport" content="width=device-width">
     <meta name="title" content="Contributing to the Knowledge Centre ">
-    <meta name="generator" content="docfx 2.46.0.0">
+    <meta name="generator" content="docfx 2.45.1.0">
```

Comment:
Why the version downgrade?

Reply:
I've used this github action

https://github.com/marketplace/actions/docfx-action?version=v0.1.0

https://github.com/nikeee/docfx-action/blob/v0.1.0/Dockerfile

Presumably when it was written 2.45.1.0 was the most recent version.

The downgrade doesn't seem to have caused any changes other than the version in the generator tag, so I don't think it's at all consequential.

But if we were to fork this github action and put a copy in the UKCloud namespace we could upgrade it and push it back to the original author.